Examples/Sudoku-examples/CLIPS-puzzles/grid3x3-p1.clp

```diff
@@ -30,12 +30,18 @@
 ;;; 
 ;;;    Naked Single
 
-
+(bind ?override "(progn
```

Comment:
This is the magic which allows defining configuration before globals have been loaded!  As a string, CLIPS will not attempt to resolve symbols.

Reply:
I have been clear in my emails that ANY change to CSP-Rules (in particular to the default config files) that would make it totally dependent on gradle and would exclude any non-programmer is unwanted. Such changes should appear in an AUTO-TEST sub-directory and only there, where expert users may want to run these tests; multiple config files for the same app is not a problem.
The examples files are not designed to be executed by CSP-Rules. But they are designed to be readable by a human user.